cpp/ixtheo_marc_pipeline.sh

```diff
@@ -2,6 +2,8 @@
 # Runs through the phases of the IxTheo MARC processing pipeline.
 set -o errexit -o nounset
 
+VUFIND_HOME=/usr/local/vufind2
```

Comment:
Isn't this inherited?  If so, I think it is dangerous to do this.  You could use the optional assignemnt that only happens if the variable does not already have a value.